tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -41,19 +42,23 @@ describe(__filename, () => {
 
     // These fields will be visible during loading since
     // they will always exist for the loaded add-on.
-    expect(root.find('.AddonMoreInfo-last-updated-title')).toHaveLength(1);
+    expect(root.find({ className: 'AddonMoreInfo-last-updated' }))
+      .toHaveLength(1);
 
     expect(root.find(LoadingText)).toHaveLength(1);
 
     // These fields will not be visible during loading
     // since they may not exist.
-    expect(root.find('.AddonMoreInfo-links-title')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-license-title')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-privacy-policy-title')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-version-title')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-version-history-title')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-eula-title')).toHaveLength(0);
-    expect(root.find('.AddonMoreInfo-beta-versions-title')).toHaveLength(0);
+    expect(root.find({ className: 'AddonMoreInfo-links' })).toHaveLength(0);
```

Comment:
All of these calls seem overly verbose. What's your objection to `root.find('.AddonMoreInfo-links')`? It seems to be working for me locally.

Reply:
I think the `toHaveLength(0)` will work regardless because it won't find either `. AddonMoreInfo-links` or `{ className: 'AddonMoreInfo-links' }`... but does it work when you are verifying something *does* exist? I'll check and if so I'll switch back.

Reply:
Indeed, using just `root.find('.Addon[...]')` worked, so I've switched back. 🤷‍♂️